packages/api-providers/src/abstracts/currency/orml-currency.ts

```diff
@@ -18,7 +18,14 @@ export class ORMLCurrency {
   constructor(private api: WebbPolkadot) {}
 
   async list() {
-    const assets = await this.api.api.query.assetRegistry.assets.entries();
+    let assets;
+
+    try {
+      assets = await this.api.api.query.assetRegistry.assets.entries();
+    } catch (ex) {
+      console.log('tried to get assets from a chain without support');
+      return [];
+    }
```

Comment:
If you can import types and get the right values instead of `toHuman()` this would be better

And  you can check for the asset registry existence

```
const assetRegistry = this.api.api.query. assetRegistry;
if(!assetRegistry) {
console.log('tried to get assets from a chain without support');
return []
}
...
```

Reply:
I'm confused as to where the 'toHuman()` call you are referring to is in my PR.